backend/chalice/upload_failures/requirements.txt

```diff
@@ -0,0 +1,8 @@
+psycopg2-binary>=2.8.5
+SQLAlchemy>=1.3.17
+requests>=2.22.0
+connexion[swagger-ui]==2.7.0
+Flask-Cors>=3.0.6
+python-jose[cryptography]>=3.1.0
```

Comment:
Are these needed?

Reply:
requests and connexion seem to be required but I was able to remove the other two